unifi/Dockerfile

```diff
@@ -8,13 +8,13 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # Setup base system
 RUN \
     apt-get update \
+    && apt-get upgrade -y \
```

Comment:
This should not be run here. This is the responsibility of the base image.

```suggestion
```

Reply:
In general I agree, in this specific case, the 8.2.0 base image we are using is 100MB and 1 year old, there's a 8.2.1 base image, but it's also 1 year old and 500MB (5 times as big).
To get newer images, we'd have to switch to a 9.0.x base image, but it did not look like a simple change.

`apt-get upgrade -y` felt like an acceptable trade-off wrt security until the add-on is moved to 9.0.x.
I'll drop the commit if you disagree.

Reply:
> To get newer images, we'd have to switch to a 9.0.x base image, but it did not look like a simple change.

I had looked at this too to go to later versions of the base image. I think the main issue is the fact that there is no package for Mongodb for Jammy. For all other main packages there are equivalents in Jammy.

Reply:
> I had looked at this too to go to later versions of the base image. I think the main issue is the fact that there is no package for Mongodb for Jammy. For all other main packages there are equivalents in Jammy.

[offtopic for this PR]

I actually gave it a try, wip branch using https://get.glennr.nl/unifi/install/unifi-8.0.26.sh for inspiration can be found at https://github.com/cfergeau/addon-unifi/tree/base9
The addon fails to start, but it's hopefully something minor, as it is working if I start the container with `--entrypoint=/bin/bash` and then run `/etc/init.d/unifi start`